### DISCUSSION TITLE
Estou com um problema com listas :(

### DISCUSSION THREAD

**Discussion by JonatasSilva01** [community/community#124411] [Programming Help]
Estou precisando de ajuda em uma lista encadeada aonde eu preciso clicar uma vez no botão e trazer 2 objetos json com uma algumas informações diferentes

**Comment by ebndev** [comment]
Hi @JonatasSilva01, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
